app/controllers/v1/interventions_controller.rb

```diff
@@ -5,28 +5,36 @@ def index
     collection = interventions_scope.detailed_search(params, current_v1_user)
     paginated_collection = V1::Paginate.call(collection, start_index, end_index)
 
-    render json: serialized_hash(paginated_collection, 'SimpleIntervention',
-                                 params: { current_user_id: current_v1_user.id }).merge({ interventions_size: collection.size }).to_json
+    data = serialized_hash(paginated_collection, 'SimpleIntervention',
+                           params: { current_user_id: current_v1_user.id })[:data]
+
+    paginated_collection.zip(data).each do |intervention, d|
+      d[:attributes][:starred] = starred?(intervention)
+    end
```

Comment:
I prefer to move this logic to the serializer or to the module in the serializer. Currently, we iterate the full collection twice instead of once